notebook/agentchat_RetrieveChat.ipynb

```diff
@@ -148,7 +148,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accepted file formats for `docs_path`:\n",
+      "['txt', 'json', 'csv', 'tsv', 'md', 'html', 'htm', 'rtf', 'rst', 'jsonl', 'log', 'xml', 'yaml', 'yml']\n"
+     ]
```

Comment:
This cell will need to be rerun after the merge to main to include `pdf` (if you want to keep it up to date).